curriculum/challenges/english/blocks/workshop-registration-form/60fab8367d35de04e5cb7929.md

```diff
@@ -7,48 +7,44 @@ dashedName: step-27
 
 # --description--
 
-Add the text `I accept the terms and conditions` immediately after the `input` element in the newly added `label`. Then link the text `terms and conditions` to the following location:
+Add an anchor `a` with the text `Read our terms and conditions` before the newly added `label`. Link the text `terms and conditions` to:
```

Comment:
```suggestion
Add an anchor element with the text `Read our terms and conditions` before the newly added `label`. Set the `href` to:
```

the whole text is inside the anchor element, so the instructions need to be adjusted, as you can't link only part of the text